Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -34,23 +34,33 @@ public class WebPage_v0 {
         WKWebView.handlesURLScheme(scheme)
     }
 
-    private init(configuration: Configuration, _navigationDecider navigationDecider: (any NavigationDeciding)?) {
-        self.configuration = configuration
+    private init(_configuration: Configuration, _navigationDecider navigationDecider: (any NavigationDeciding)?, _dialogPresenter dialogPresenter: (any DialogPresenting)?) {
```

Comment:
Curious about the `_`s in an already `private` method? Is this to avoid ambiguous overloads?

Reply:
ya, the initializers beneath it caused this to become ambiguous